package.json

```diff
@@ -99,6 +99,7 @@
     "npm-run-all": "^4.1.5",
     "nyc": "^14.1.1",
     "puppeteer": "^1.20.0",
+    "request": "^2.88.0",
```

Comment:
Was this intentional?

Reply:
Yes, it's needed for tests (peerDependency of "request-promise-native")